requirements.txt

```diff
@@ -1,4 +1,12 @@
 pyquery
 demjson
 requests
-selenium
\ No newline at end of file
+selenium
+Scrapy==1.3.3
```

Comment:
Can we avoid this dependency? If it's just for the selector, can we use PyQuery (already a requirement) instead?